test/sslapitest.c

```diff
@@ -11151,6 +11157,12 @@ static int test_tls13_record_padding(int idx)
             goto end;
         if (!TEST_true(SSL_set_block_padding(clientssl, 512)))
             goto end;
+    } else if (idx == 5) {
+        /* pad server and client handshake only */
```

Comment:
Should we attempt to pad beyond the max permitted size like we do for `idx == 3`

Reply:
sure, added that